src/ui/ExportMATFunctions.jl

```diff
@@ -1,25 +1,24 @@
 function export_2_mat_sequence(seq, matfolder; matfilename="seq_sequence.mat")
 	max_rf_samples=100
     N = length(seq)
-    ΔT = KomaMRICore.durs(seq)
-    T0 = cumsum([0; ΔT],dims=1)
+    T0 = get_block_start_times(seq)
     off_val = Inf #This removes the unnecessary points in the plot
 
     #GRADS
-    t1x = vcat([KomaMRICore.get_theo_t(seq.GR[1,i]) .+ T0[i] for i=1:N]...)
-    t1y = vcat([KomaMRICore.get_theo_t(seq.GR[2,i]) .+ T0[i] for i=1:N]...)
-    t1z = vcat([KomaMRICore.get_theo_t(seq.GR[3,i]) .+ T0[i] for i=1:N]...)
-    Gx =  vcat([KomaMRICore.get_theo_A(seq.GR[1,i];off_val) for i=1:N]...)
-    Gy =  vcat([KomaMRICore.get_theo_A(seq.GR[2,i];off_val) for i=1:N]...)
-    Gz =  vcat([KomaMRICore.get_theo_A(seq.GR[3,i];off_val) for i=1:N]...)
+    t1x = vcat([KomaMRIBase.get_theo_t(seq.GR[1,i]) .+ T0[i] for i=1:N]...)
+    t1y = vcat([KomaMRIBase.get_theo_t(seq.GR[2,i]) .+ T0[i] for i=1:N]...)
+    t1z = vcat([KomaMRIBase.get_theo_t(seq.GR[3,i]) .+ T0[i] for i=1:N]...)
+    Gx =  vcat([KomaMRIBase.get_theo_A(seq.GR[1,i];off_val) for i=1:N]...)
+    Gy =  vcat([KomaMRIBase.get_theo_A(seq.GR[2,i];off_val) for i=1:N]...)
+    Gz =  vcat([KomaMRIBase.get_theo_A(seq.GR[3,i];off_val) for i=1:N]...)
     GRADS = hcat(t1x, t1y, t1z, Gx, Gy, Gz)
     #RFS
-    t2 =  vcat([KomaMRICore.get_theo_t(seq.RF[1,i];max_rf_samples) .+ T0[i] for i=1:N]...)
-    R =   vcat([KomaMRICore.get_theo_A(r;off_val,max_rf_samples) for r = seq.RF]...)
+    t2 =  vcat([KomaMRIBase.get_theo_t(seq.RF[1,i];max_rf_samples) .+ T0[i] for i=1:N]...)
+    R =   vcat([KomaMRIBase.get_theo_A(r;off_val,max_rf_samples) for r = seq.RF]...)
     RFS = hcat(t2, R)
     #ADC
-    t3 =  vcat([KomaMRICore.get_theo_t(seq.ADC[i])  .+ T0[i] for i=1:N]...)
-    D =   vcat([KomaMRICore.get_theo_A(d;off_val) for d = seq.ADC]...)
+    t3 =  vcat([KomaMRIBase.get_theo_t(seq.ADC[i])  .+ T0[i] for i=1:N]...)
+    D =   vcat([KomaMRIBase.get_theo_A(d;off_val) for d = seq.ADC]...)
```

Comment:
Repeated code. Already used in plots. Put in function.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/c64abd9ef7372e27a967fd2d8357eebadb724a72